README.md

```diff
@@ -57,6 +57,22 @@ Programatically send AWS usage report to Zulip.
    $ python3 report.py --aws-profile bot
    ```
 
+## Environmental variables
+
+You can overwrite each config by the following environmental variables.
+
+* `AWS_ACCOUNT_ID`
+* `ZULIP_SITE`
+* `ZULIP_EMAIL`
+* `ZULIP_API_KEY`
+* `ZULIP_TYPE`
+* `ZULIP_TO`
+* `ZULIP_TOPIC`
+* `ZULIP_MESSAGE`
```

Comment:
It might be obvious, but it is a good way to explain examples here.
ZULIP_TYPE = stream ZULIP_TO = (stream name) and so on.

Reply:
Thanks!
I added the table!